lib/assessment.rb

```diff
@@ -5,35 +5,52 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
+  attr_reader :name
+  attr_reader :breed
+  attr_reader :age
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
 
 # Question 2
 # Create a new Dog object.
 
-Answer.dog = nil # replace nil with your code
+Answer.dog = Dog.new( "Max", "Lab", "9")
 
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
-##
-# your code here
-##
+class Person
+  attr_accessor :name
+  attr_accessor :location
+  attr_reader :age
+  def initialize(name, age, location)
+    @name = name
+    @age = age
+    @location = location
+  end
+end
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
 
-Answer.dave = nil # replace nil with your code
-Answer.dave.location = nil # replace nil with your code
+Answer.dave = Person.new("Dave", 32, "Somerville")
+Answer.dave.location = "Somerville"
 
 # Question 5
 # Create another class called Developer that inherits from Person.
 # Give it a new public method called 'hire_for_job'
 
-##
-# your code here
-##
+class Developer
+#   Hires = []
+#   Developer << Person
+#   def hire_for_job
+#     Hires << Person
+end
```

Comment:
To make a class inherit from another class, you use the left carrot, `<`. 
e.g.
`class Developer < Person`

This gives `Developer` access to all of `Person`'s methods.

The `hire_for_job` method can be any method you like, as long as it's unique to `Developer` (i.e. `Person` _can't_ use it).